src/org/zaproxy/zap/extension/pscanrulesAlpha/CacheableScanner.java

```diff
@@ -597,8 +597,8 @@ public void alertNonStorable (HttpMessage msg, int id, String evidence) {
 				Constant.messages.getString(MESSAGE_PREFIX_NONSTORABLE + "soln"), //Solution
 				Constant.messages.getString(MESSAGE_PREFIX_NONSTORABLE + "refs"), //References
 				evidence,	// Evidence
-				0,	// 0 - there is no CWE for non-cachable contents
-	            0,	// 0 - there is no WASC for non-cachable contents 
+				525, // CWE-525: Information Exposure Through Browser Caching
```

Comment:
Better with 524 ("Information Exposure Through Caching")? This information alert does not apply only to browser caching.